nix/container/vpn/strongswan.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/sh
 
-# The charon binary is not included in the PATH generated by nixery.dev, find it manually.
+set -eux
```

Comment:
Is the -x intended? Feel free to keep.

Reply:
Yes, `-x` is currently set in the deployment and only moves to the script here. I wanted to have some logs for what happens before charon is ready, and `-x` is the poor man's solution for that :)